networkinglab/src/Components/Cards.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+const styles = {
+    img: {
+        height: "200px",
+        width: "300"
+    }
+}
+const Cards =({img})=>{
+    return (
+        <img src ={img} style = {styles.img} alt =""></img>
```

Comment:
In most cases we want to stay away from using in-line styling. We want to have all of our styling as on the CSS files